go.mod

```diff
@@ -76,6 +76,7 @@ require (
 	github.com/juju/terms-client/v2 v2.0.0
 	github.com/juju/testing v1.2.0
 	github.com/juju/txn/v3 v3.0.2
+	github.com/juju/utils/v3 v3.1.0
```

Comment:
We should only be using utils/v4

Reply:
Yep, wasn't me. Still being brought in by an indirect dependency.  I ran go mod tidy again to get the comment in.